qutebrowser/components/utils/blockutils.py

```diff
@@ -62,17 +63,16 @@ class BlocklistDownloads:
         _finished: Has `_user_cb_all` been called?
     """
 
+    single_download_finished = pyqtSignal(object)  # arg: the file object
```

Comment:
For some reason, PyQT doesn't like it when I make this type `io.IOBase`.

Reply:
I can imagine, it's a bit picky about the types listed there and I'm not sure what the "internal" Python type for a file object is exactly (those IO types are a bit weird). Seems fine to use `object` instead.